style.css

```diff
@@ -184,3 +184,13 @@ h6 {
   background-color: rgb(203, 216, 203);
   box-shadow: 0px 0px 3px rgba(0,0,0,0.3);
 }
+
+.expandableArea {
+  clear: both;
+  display: none;
+}
+
+.expanded > .expandableArea {
+  display: inline !important;
```

Comment:
Is the `!important` still necessary here? I thought we just added that for testing?